wallet/createtx.go

```diff
@@ -1362,8 +1363,10 @@ func (w *Wallet) purchaseTickets(ctx context.Context, op errors.Op,
 	// do so now.
 	var vspFee dcrutil.Amount
 	if poolAddress != nil {
+		// poolAddress is only used with the legacy stakepool
+		const dcp0010Active = false
 		vspFee = txrules.StakePoolTicketFee(ticketPrice, ticketFee,
-			tipHeight, poolFees, w.ChainParams())
+			tipHeight, poolFees, w.ChainParams(), dcp0010Active)
```

Comment:
Hm, thinking about legacy dcrstakepool deployments. With this hard-coded `false`, neither client nor server are going to be updated to the new fee estimation. Assuming dcrstakepool continues to work after the new agendas activate (not guaranteed), the server operators will continue receiving the old fees, they will not benefit from the new increased vote rewards.

Reply:
Sorry @jrick, I just realised that you mentioned this in your PR description. Looks like we are on the same page and agree this is not worth spending time on.

Reply:
Resolved